src/service/standard-actions-impl.js

```diff
@@ -311,17 +323,36 @@ export class StandardActions {
       }
     });
 
-    this.resources_.mutateElement(target, () => {
-      if (target.classList.contains('i-amphtml-element')) {
-        target./*OK*/expand();
-      } else {
-        toggle(target, true);
-      }
-    });
+    const autofocusElOrNull = getAutofocusElementForShowAction(target);
+
+    // Safari only honors focus in sync operations.
+    if (autofocusElOrNull && Services.platformFor(ownerWindow).isSafari()) {
```

Comment:
Check for iOS instead, it is an iOS thing (I don't think isSafari catches Chrome on iOS for example)

Reply:
Gotcha, fixed. PTAL.